vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import react from "@vitejs/plugin-react";
+import {defineConfig} from "vite";
+
+
+// https://vite.dev/config/
+export default defineConfig({
+    appType: "mpa",
+    base: "./",
```

Comment:
If not specified, after build, the log viewer can only be served at a website root.